src/Apps/Artwork/Components/ArtworkChatBubble.tsx

```diff
@@ -39,12 +39,15 @@ const ArtworkChatBubble: FC<ArtworkChatBubbleProps> = ({ artwork }) => {
     return null
   }
 
-  return getENV("SALESFORCE_CHAT_ENABLED") ? (
+  if (!getENV("SALESFORCE_CHAT_ENABLED")) {
+    return null
+  }
```

Comment:
Thanks for removing the ternary here. Quick early exits (no matter how many there are, one after the other) make code way more readable 👍

Reply:
Thank you actually, I only did this because I saw you [doing it](https://github.com/artsy/force/pull/12572/commits/9cda3c85384f255b3399f0509cd021989cd23cec) and it really seemed much easier to read ❤️